contrib/cscope/packages.el

```diff
@@ -0,0 +1,60 @@
+;;; packages.el --- cscope Layer packages File for Spacemacs
+;;
+;; Copyright (c) 2012-2014 Sylvain Benner
+;; Copyright (c) 2014-2015 Sylvain Benner & Contributors
+;;
+;; Author: Sylvain Benner <sylvain.benner@gmail.com>
+;; URL: https://github.com/syl20bnr/spacemacs
+;;
+;; This file is not part of GNU Emacs.
+;;
+;;; License: GPLv3
+
+(setq cscope-packages '(evil-jumper
+                        helm-cscope
+                        xcscope))
```

Comment:
That's not what I'm seeing. Just deleted both `helm-cscope` and `xcscope` from my elpa directory and restarted Emacs -  Spacemacs installed them automatically.